### ISSUE TITLE
Termination stati missing MOI equivalents

### ISSUE THREAD

**Issue by inversed-ru** [oxfordcontrol/Clarabel.jl#129]
`ALMOST_PRIMAL_INFEASIBLE` and `ALMOST_DUAL_INFEASIBLE` termination stati are missing their MOI counterparts in `ClarabeltoMOITerminationStatus` (`MOI_Wrapper.jl`), which sometimes causes errors like `KeyError: key Clarabel.ALMOST_PRIMAL_INFEASIBLE not found`. The fix is trivial: add
```
Clarabel.ALMOST_PRIMAL_INFEASIBLE   =>  MOI.ALMOST_INFEASIBLE,
Clarabel.ALMOST_DUAL_INFEASIBLE     =>  MOI.ALMOST_DUAL_INFEASIBLE
```
to `ClarabeltoMOITerminationStatus` Dict.

**Comment by inversed-ru** [comment]
`ClarabeltoMOIPrimalStatus` and `ClarabeltoMOIDualStatus` dicts are missing these stati as well.

**Comment by goulart-paul** [comment]
Fixed by #132.
